.github/workflows/prepare-release.yml

```diff
@@ -33,3 +35,58 @@ jobs:
           publish: "npm run release"
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  release:
+    name: Deploy and release to sandbox
+    needs: changesets
+    if: needs.prepare.outputs.published == 'true'
+    runs-on: ubuntu-latest
+    env:
+      API_URI: /graphql/
+      APP_MOUNT_URI: /dashboard/
+      STATIC_URL: /dashboard/static/
+      SENTRY_ORG: saleor
+      SENTRY_PROJECT: dashboard
+      SENTRY_URL_PREFIX: "~/dashboard/static"
+      SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
+      SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+      APPS_MARKETPLACE_API_URI: "https://apps.staging.saleor.io/api/v2/saleor-apps"
+      VERSION: ${{ github.event.inputs.git_ref || github.ref_name }}
+      IS_CLOUD_INSTANCE: true
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: ".nvmrc"
+      - name: Install Dependencies
+        run: npm ci
+      - name: Build
+        run: |
+          npm run build
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
```

Comment:
I suggest upgrading to v4 as v1 uses deprecated features and runtime. But it's a bigger issue as it needs to be ported all the way down to v3.11, up to you if you want to do it in this PR or another PR

Reply:
change it here, but in terms of other versions will do separate PRs